code/game/machinery/vending.dm

```diff
@@ -2427,3 +2464,17 @@ var/global/num_vending_terminals = 1
 	for(var/random_items = 1 to premium.len - 4)
 		premium.Remove(pick(premium))
 	src.initialize()
+
+
+/obj/machinery/vending/sale
+	name = "Sales"
+	desc = "Buy, sell, repeat."
+	icon_state = "sale"
+	//vend_reply = "Insert another joke here"
+	//product_ads = "Another joke here"
+	//product_slogans = "Jokes"
+	products = list(
+		/obj/item/clothing/shoes/black = 10,
```

Comment:
They're a placeholder in case someone spawns this vending machine as an admin (nobody should ever do it)